tests/orion/services/test_flow_run_notifications.py

```diff
@@ -99,6 +100,24 @@ async def test_service_sends_notifications(
     )
 
 
+@pytest.mark.parametrize(
+    "api_url,ui_url",
+    [
+        (None, "http://ephemeral-orion/api"),
+        ("https://api.prefect.cloud/api", "https://app.prefect.cloud"),
+        ("http://my-orion/api", "http://my-orion/"),
+    ],
+)
+def test_get_ui_url_for_flow_run_id(flow_run, api_url, ui_url):
+    with temporary_settings({"PREFECT_API_URL": api_url}):
```

Comment:
```suggestion
    with temporary_settings({PREFECT_API_URL: api_url}):
```
You have the pass in the actual setting object as opposed to a string, for `temporary_settings`

Reply:
Do these tests need to be updated to use the `UI_URL` now? Or are we relying on inference?

Reply:
We rely on inference. If we set the `UI_URL` it will not do validation:
```python
def default_ui_url(settings, value):
    if value is not None:
        return value
     ...
     if ui_url.endswith("/api"):
     # Handles open-source APIs
     ui_url = ui_url[:-4]

      # Handles Cloud APIs with content after `/api`
      ui_url = ui_url.replace("/api/", "/")
```

Reply:
@madkinsz let me know if you're cool with this or if we need to make additional changes to the logic. It should be ready otherwise.